src/components/SecondaryNav/SecondaryNavHeader/SecondaryNavHeader.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'clsx';
+import styles from './SecondaryNavHeader.scss';
+
+const SecondaryNavHeader = ({children}) => {
+    return (
+        <div className={classnames('.flexCol_center', '.alignCenter', styles.secondaryNavHeader)}>
```

Comment:
- You don't need the dot before classes
- I think is better to give a row context in case we want to align icon and text, what do you think?
```suggestion
        <div className={classnames('flexRow_center', 'alignCenter', styles.secondaryNavHeader)}>
```